templates/terraform/resource.erb

```diff
@@ -26,7 +26,7 @@ package google
     settable_properties = properties.reject{ |v| v.output && !v.is_a?(Api::Type::Fingerprint) }.reject(&:url_param_only)
     # PUT needs parameters like `name` to be set in the resource body, but we don't want to send them in PATCH
     update_body_properties = settable_properties
```

Comment:
This line can be removed now

Reply:
Not sure I follow - this is making sure `update_body_properties` is the same as `settable_properties` when the update verb is `:PUT`

Reply:
ah ok, that if afterwards screwed me up. i would think an if/else would be easier to follow but idk how rubocop feels about that

Reply:
I *think* this is what rubocop would most want, but who knows. Using an if/else means we need to pull out the variable, and I'd like to avoid a ternary assignment here.